src/test/java/lesson1/SalaryTest.java

```diff
@@ -0,0 +1,38 @@
+package lesson1;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SalaryTest {
+
+    private Salary salary;
+
+    @Before
+    public void setup(){
+        salary=new Salary();
+    }
+
+    @Test
+    public void itCalculateMonthSalary() {
+        int actual=salary.calculateMonthSalary(-200,168);
```

Comment:
We need to use `@Test(expected= MyException.class)` to test unhappy cases.

Reply:
changes added